pkg/controllers/chart/chart.go

```diff
@@ -1,4 +1,4 @@
-package helm
+package chart
```

Comment:
I'm curious about the rationale for this package move - the pattern for other projects that use Wrangler is to put the controller in `pkg/foo/controller.go`, for example system-upgrade-controller has its controller in package `upgrade` at  [`pkg/upgrade/controller.go`](https://github.com/rancher/system-upgrade-controller/blob/master/pkg/upgrade/controller.go)

Reply:
Is this in anticipation of having multiple controllers?

Reply:
Yes, the structure here follows what Fleet and Rancher do today. Example for Fleet: https://github.com/rancher/fleet/tree/master/pkg/controllers

Reply:
More importantly, this opens the door for moving the registration logic for controllers into a separate package instead of putting it in main.go and putting constants in a separate package.

i.e. the final structure looks like this: https://github.com/aiyengar2/helm-controller/tree/master/pkg/controllers

Reply:
Where the content that used to live in main.go is moved to the controllers package in https://github.com/aiyengar2/helm-controller/blob/e8055c48bf2cd3bc3bed5f311b501daf599ce28f/pkg/controllers/controllers.go#L37-L201

Reply:
thanks for the explanation. Sound like we might want to reorg system-upgrade-controller too at some point, if we're going to continue to maintain it.